doc/start/hardware-recommendations.rst

```diff
@@ -169,29 +184,43 @@ performance of sequential reads and writes.
    SSD in a test configuration to gauge performance. 
 
 Relatively inexpensive SSDs may appeal to your sense of economy. Use caution.
-Acceptable IOPS are not enough when selecting an SSD for use with Ceph. 
+Acceptable IOPS are not the only factor to consider when selecting an SSD for
+use with Ceph. 
 
-SSDs have historically been cost prohibitive for object storage, though
-emerging QLC drives are closing the gap.  HDD OSDs may see a significant
+SSDs have historically been cost prohibitive for object storage, but 
+emerging QLC drives are closing the gap. HDD OSDs may see a significant
```

Comment:
`closing or crossing the gap, offering greater density with lower power/cooling.  `

Maybe this would be a good place for the SNIA TCO calc link.

Reply:
Done (not verbatim, but close).

And the link is there now, too. That's the kind of thing I'm keen to attach to a core document like this.